lib/index.js

```diff
@@ -1151,17 +1159,17 @@ module.exports = (function() {
             if(self.option && self.option.imageRatio){
                 ratio = self.option.imageRatio;
             }							
-            imageWidth = Number.parseInt(imageWidth / ratio);
-            imageHeight = Number.parseInt(imageHeight / ratio);
+            imageWidth = Math.floor(imageWidth / ratio);
+            imageHeight = Math.floor(imageHeight / ratio);
```

Comment:
@jdugh Notice these changes. You likely would be affected by this PR if you continue without taking into account this work. These changes and other similar changes below was brought up by TSC, and I think it slightly simplify writing code. Let me know if I made some mistakes during removal.